scripts/generate_compile_commands.py

```diff
@@ -19,7 +19,7 @@
 
 # These work for node-cpp-skel to detect the files being compiled
 # They may need to be modified if you adapt this to another tool
-matcher = re.compile('^(.*) (.+cpp)\n')
+matcher = re.compile('^(.*) (.+cpp)')
```

Comment:
Some format had changed in the output which is regexped, so matcher had to be changed.